include/finufft/spreadinterp.h

```diff
@@ -30,33 +30,26 @@ enum {
 namespace finufft {
 namespace spreadinterp {
 
-// things external (spreadinterp) interface needs...
 template<typename T>
-FINUFFT_EXPORT int FINUFFT_CDECL spreadinterp(
-    UBIGINT N1, UBIGINT N2, UBIGINT N3, T *data_uniform, UBIGINT M, T *kx, T *ky, T *kz,
-    T *data_nonuniform, const finufft_spread_opts &opts);
-
-FINUFFT_EXPORT int FINUFFT_CDECL spreadcheck(UBIGINT N1, UBIGINT N2, UBIGINT N3,
-                                             const finufft_spread_opts &opts);
-template<typename T>
-FINUFFT_EXPORT int FINUFFT_CDECL indexSort(std::vector<BIGINT> &sort_indices, UBIGINT N1,
-                                           UBIGINT N2, UBIGINT N3, UBIGINT N, T *kx,
-                                           T *ky, T *kz, const finufft_spread_opts &opts);
-template<typename T>
-FINUFFT_EXPORT int FINUFFT_CDECL spreadinterpSorted(
-    const std::vector<BIGINT> &sort_indices, const UBIGINT N1, const UBIGINT N2,
-    const UBIGINT N3, T *data_uniform, const UBIGINT M, T *FINUFFT_RESTRICT kx,
-    T *FINUFFT_RESTRICT ky, T *FINUFFT_RESTRICT kz, T *FINUFFT_RESTRICT data_nonuniform,
-    const finufft_spread_opts &opts, int did_sort, bool adjoint);
+FINUFFT_EXPORT int spreadinterp(UBIGINT N1, UBIGINT N2, UBIGINT N3, T *data_uniform,
```

Comment:
Would FINUFFT_EXPORT_TEST be enough here? I think that `spreadinterp` is not part of the public interface.